### PR TITLE
ENH: Added cosine data seed to mds

### LINKED ISSUE #13: Add a test exercising run headers in fake MDS
Create a test function that inserts events, event descriptors, and run headers. An example doing the first two is already in the notebook, but run headers haven't been addressed at all.

Bonus points if this also works in live MDS, but if that's any trouble let's wait until this latest revision is written and merged.

### PR DESCRIPTION
**Pull request by ericdill**
Closes #13

### CODE REVIEW

File: examples/seeding_data/seed_mds/synthetic_cosine.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+for idx, i in enumerate(np.linspace(start, stop, num)):
+    data = {'linear_motor': i,
+            'Tsam': i + 5,
+            'scalar_detector': func(i)}
+    e = save_event(header=h, event_descriptor=e_desc, seq_no=idx,
+                   beamline_id='csx', timestamp=time.time(),
+                   data=data)
+    time.sleep(sleep_time)
+
+print find_last()
```
Comment (danielballan) [inline examples/seeding_data/seed_mds/synthetic_cosine.py:46]: Better make this py2/3 safe.

File: examples/seeding_data/seed_mds/synthetic_cosine.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+from metadataStore.api.collection import (save_header, save_beamline_config,
+                                          save_event, save_event_descriptor)
+from metadataStore.api.analysis import find_last
+import random
+import time
+import string
+import numpy as np
+
+def id_generator(size=6, chars=string.ascii_uppercase + string.digits):
```
Comment (tacaswell) [inline examples/seeding_data/seed_mds/synthetic_cosine.py:9]: Use https://docs.python.org/2/library/uuid.html#uuid.uuid4 instead.

**Comment by danielballan** [comment]
Perfect. See line remark for a minor revision. Also, could you put a module docstring at the top explaining this in a sentence or two?

**Comment by ericdill** [comment]
I'm turning this into a more interesting data seeder

**Comment by danielballan** [comment]
For the record, this MDS-specific example is going into MDS via arkilic/metadataStore#1

**Pull request closed without merging**